src/MoBi.Presentation/Tasks/Interaction/ObjectBaseTask.cs

```diff
@@ -1,128 +0,0 @@
-﻿using System.Collections.Generic;
```

Comment:
A lot of the changed files here are because I removed this task. It mostly renamed objects and up to now, all the renaming was completely common for all building blocks.

I needed to rename building blocks now in a type-sensitive way because the renaming presenter for Expression Profile would be different from most other cases.

Ultimately, this code was added to one of the base 'Interaction' tasks rather than delegated to this type-insensitive class.

There were also a couple of dependencies that needed to be injected as a result.